packages/networks/src/networks/celo.ts

```diff
@@ -40,6 +40,11 @@ export const celo: NetworkConfig = {
   previousDeploys: [],
   isTestNetwork: false,
   teamMultisig: '0xc293E2da9E558bD8B1DFfC4a7b174729fAb2e4E8',
+  uniswapV3: {
```

Comment:
let's only use official subgraphs... built by the Uniswap team. This is obviously _very_ sensitive and we don't want to rely on data we can't reliably trust.